lib/WorkflowMain.groovy

```diff
@@ -80,9 +82,8 @@ class WorkflowMain {
         NfcoreTemplate.awsBatch(workflow, params)
 
         // Check input has been provided
-        if (!params.input) {
-            log.error "Please provide an input samplesheet to the pipeline e.g. '--input samplesheet.csv'"
-            System.exit(1)
+        if (params.analysis == 'targeted' && !params.input) {
```

Comment:
Is this not also needed for the other case`if (params.analysis == `screening` && !params.input)` ?

Reply:
@LaurenceKuhl was there a reason why input is not mandatory for screening? I think we are anyways checking it with the JSON schema.

Reply:
I removed the if for "targeted" :)